src/js/wrap.cpp

```diff
@@ -954,6 +1028,16 @@ namespace js
           "Failed to register JS class definition for node");
       }
     }
+
+    // Register class for node
+    {
+      auto ret = JS_NewClass(rt, network_class_id, &network_class_def);
```

Comment:
Should we just go through `NodeState` to access the ledger secrets, instead of registering another class?

Reply:
That seemed awkward because:

1. the abstract node state doesn't expose the network state, which seems deliberate
2. they're both scoped at the same level under Enclave

But I also don't have strong feelings about it, I could be convinced otherwise.